README.md

```diff
@@ -12,64 +12,157 @@ A resource may classify its messages into groupings,
 and it may include additional data or metadata relating to them.
 It should be possible to represent a resource in a text-based human-friendly manner.
 
-### How to Contribute
-
-- **Issues** are for specific aspects of the resulting specification.
-  Discussions here should be focused, and stay on topic.
-  New issues must include a sufficient initial description to introduce themselves,
-  and may contain an exploration of possible solutions to them.
-- **Pull requests** help develop and define the eventual specification.
-  Accepting and merging a PR should not be taken as the final word on any topic,
-  but each change should be an improvement on the previous.
-  Each PR should only apply one change that may be squashed to the `main` branch.
-  Text content should use [semantic line breaks](https://sembr.org/).
-  A PR may close one or more issues, but this is not required.
-- **Discussions** are freeform, and may well refer to multiple issues or topics.
-  This is also the forum in which we should develop further our ways of working,
-  which may also include interactions and forums outside this repository,
-  such as occasional video calls.
-
-We welcome your participation and interest!
-
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The prior work on a new message format has identified the following challenges
+that go beyond or arise from defining the syntax and behavior of a single message,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 message syntax is naturally multi-line,
+  and multi-line values are not easy to use in many of the current resource formats.
+- Comments and metadata are important for translators,
+  but (mostly) not relevant for formatting.
+  Their attachment to messages needs to be well specified,
+  while being easy to read, write, and ignore.
+- Structured message metadata needs a common schema to be universally understood.
+- Many messages are composed of multiple localizable parts
+  (such as an HTML element with a localizable body and localizable attributes),
```

Comment:
Again, I'd say "resource" here to avoid confusion with MF2 _message_... although you might intend _message_ here, e.g.:

```
.local $name = {|Your Name|}
{{Your name is {#html:input type=text value=$name /}.}} 
```

Reply:
I do mean message here, and the difficulties that arise e.g. when translating your example to a language where the "Your" within the literal needs to vary depending on the person's gender.

This is solvable in MF2 by making the message as a whole vary, but then it's not possible to have a single `$name` value as above, and it arguably gets a bit hacky.

I'll add a clarification at the end of this item about this.

Reply:
I agree that my example is kind of sh%ty, but a resource system might solve it differently than MF2 does:

```
@type=multipart
@comment All my parts go together :-)
myMessage = {
   value = |Your Name|
   altText = |Foo|
   url = ...mumble....
   message = {#html:img url=$url alt=$altText/} Your name is {#html:input type=text value=$value /}.
}

Reply:
Yeah, something like that is one potential solution. I don't think there's necessarily a better word than "message" to describe something like your `myMessage` value, though.

Reply:
Resource? Imagine that the file contains a base64 encoded .ico? Or an external file reference? Probably better to resolve this comment. Color selection for the bikeshed can wait.

Reply:
How about this?
```suggestion
- Many localizable messages need to be composed together
  (such as an HTML element with a localizable body and localizable attributes),
```